index.js

```diff
@@ -202,7 +203,7 @@ function normalizeArguments(url, opts) {
 	);
 
 	opts.headers = Object.assign({
-		'user-agent': 'https://github.com/sindresorhus/got',
+		'user-agent': `sindresorhus-got/${version} (https://github.com/sindresorhus/got)`,
```

Comment:
Actually, since we include the URL, we can remove `sindresorhus-` from the first part.